packages/compass-shell/src/components/compass-shell/tab-compass-shell.tsx

```diff
@@ -0,0 +1,219 @@
+import { connect } from 'react-redux';
+import React, {
+  useCallback,
+  useEffect,
+  useRef,
+  createRef,
+  useState,
+} from 'react';
+import { useTabState } from '@mongodb-js/compass-workspaces/provider';
+import {
+  Banner,
+  Link,
+  css,
+  getScrollbarStyles,
+  palette,
+  spacing,
+} from '@mongodb-js/compass-components';
+import type { MapStateToProps } from 'react-redux';
+import type { Shell as ShellType } from '@mongosh/browser-repl';
+import type { WorkerRuntime } from '@mongosh/node-runtime-worker-thread';
+
+import ShellInfoModal from '../shell-info-modal';
+import ShellHeader from '../shell-header/shell-header';
+import type { HistoryStorage } from '../../modules/history-storage';
+import type { RootState } from '../../modules';
+import { usePreference } from 'compass-preferences-model/provider';
+
+// The browser-repl package.json defines exports['.'].require but not .module, hence require() instead of import
+const { Shell } =
+  // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/consistent-type-imports
+  require('@mongosh/browser-repl') as typeof import('@mongosh/browser-repl');
```

Comment:
This is just copied over from the existing shell component and not a new thing, right?